privacyidea/lib/realm.py

```diff
@@ -175,61 +177,57 @@ def delete_realm(realmname):
 
 
 @log_with(log)
-def set_realm(realm, resolvers=None, priority=None):
+def set_realm(realm, resolvers=None):
     """
     It takes a list of resolvers and adds these to the realm.
     If the realm does not exist, it is created.
     If the realm exists, the old resolvers are removed and the new ones
     are added.
 
-    :param realm: an existing or a new realm
-    :param resolvers: names of resolvers
+    :param realm: name of an existing or a new realm
+    :type realm: str
+    :param resolvers: list with names and options of resolvers
```

Comment:
We should define in the docstring, how the resolvers list dict looks like

[ {'name': <resolvername1>,   
   'node': <uuid of the node/optional>,
    'priority': <priority of the resolver in the realm>},
  {'name': <resolvername2>,
  }
]

Reply:
Done in 120dba6c3